Oleg branch sprint4

olegklyufinskyy opened the pull request:
Update google font

olegklyufinskyy committed:
add google font and FontSizeInput now you can chan
ge by click

olegklyufinskyy committed:
Merge branch 'oleg-branch-sprint4' of https://github.com/chingu-voyages/v47-tier2-team-17 into oleg-branch-sprint4

olegklyufinskyy commented:
Update

olegklyufinskyy committed:
tailwind update

olegklyufinskyy committed:
update FontStyleInput

joekotvas commented on fetools-app/src/pages/FontVisualizer.jsx:
```diff
@@ -154,20 +153,20 @@ const FontVisualizer = () => {
-        <HtmlCodeGenerator generateHtmlCode={generateHtmlCode} />
       </div>
     </main>
   );
```
```suggestion
  );
  
```
Bump to induce deploy preview.

joekotvas committed:
Bump to induce deploy preview

joekotvas committed:
Add Google font loading capability.

Update FontNameInput.jsx to load the selected font and add it to the document on input change.

joekotvas committed:
Fix for https restriction

Font files have to be loaded over https.

joekotvas committed:
Merge branch 'development' into oleg-branch-sprint4

joekotvas commented on fetools-app/src/pages/FontVisualizer.jsx:
Some preferred defaults:

```javascript
  const [font, setFont] = useState({
    name: 'Times New Roman',
    color: '#ffffff',
    style: 'normal',
    weight: 'bold',
    textTransform: 'none',
    textAlign: 'center',
    letterSpacing: 0,
    lineHeight: 1.2,
    fontSize: 5,
  });

  const [backgroundColor, setBackgroundColor] = useState('#222222');
```

joekotvas commented on fetools-app/src/components/FontVisualizer/Preview.jsx:
Preview text should read "Click to edit" and should be editable onClick instead of onDoubleClick

joekotvas reviewed (approved):
Great work, Oleg!

I went ahead and added font loading to FontNameInput.jsx so that the selected font will work in the preview area. There ae some notes as well.
